airbyte-cdk/java/airbyte-cdk/core/src/main/kotlin/io/airbyte/cdk/integrations/util/ConnectorExceptionUtil.kt

```diff
@@ -49,6 +55,8 @@ object ConnectorExceptionUtil {
             RECOVERY_CONNECTION_ERROR_MESSAGE
         } else if (isUnknownColumnInFieldListException(e)) {
             e!!.message
+        } else if (isTransientError(e)) {
+            DATABASE_CONNECTION_ERROR
```

Comment:
Is transient always connection error going forward?

Reply:
For now, yes. We can always tweak the setting going forward